Code/autopkglib/github/__init__.py

```diff
@@ -163,8 +164,9 @@ def search_for_name(
         results_limit: int = 100,
     ):
         """Search GitHub for results for a given name."""
-
+        name = quote(name)
         query = f"q={name}+extension:recipe+user:{user}"
```

Comment:
No reason to make a temporary variable, just stick this straight into the query:
```
query = f"q={quote(name)}+extension:recipe+user:{user}"
```

Reply:
https://github.com/autopkg/autopkg/pull/664/commits/f6b75c56365b97fb91c6762d7b780e85c4b8ca91

Reply:
Tested and working as expected